# Rebrand the product name to "Europium" (personal build).
# Changes PRODUCT_(FULL|SHORT)NAME + installer names and the macOS bundle id.
# COMPANY_* and COPYRIGHT are deliberately left as "The Chromium Authors" so the
# derivative's upstream attribution stays accurate.
# Authored against Chromium 150.0.7871.46.
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@ COMPANY_SHORTNAME=The Chromium Authors
 COMPANY_FULLNAME=The Chromium Authors
 COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Chromium
-PRODUCT_SHORTNAME=Chromium
-PRODUCT_INSTALLER_FULLNAME=Chromium Installer
-PRODUCT_INSTALLER_SHORTNAME=Chromium Installer
+PRODUCT_FULLNAME=Europium
+PRODUCT_SHORTNAME=Europium
+PRODUCT_INSTALLER_FULLNAME=Europium Installer
+PRODUCT_INSTALLER_SHORTNAME=Europium Installer
 COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
-MAC_BUNDLE_ID=org.chromium.Chromium
+MAC_BUNDLE_ID=com.fx.europium
 MAC_CREATOR_CODE=Cr24
 MAC_TEAM_ID=
